index.html

```diff
@@ -21,160 +21,232 @@
         <a class="hamburger" href="#">
```

Comment:
Kindly, I suggest you indicate an appropriate **title** for your site

Reply:
@mgmediaweb Thank you! I think I already fixed all the issues, I don't know if the appropriate title which you said is about the meta title of the site, but I fixed that too.